salt/modules/aptpkg.py

```diff
@@ -2211,6 +2216,9 @@ def expand_repo_def(**kwargs):
                                                  dist)
 
     source_entry = sourceslist.SourceEntry(repo)
+    for list_args in ('architectures', 'comps'):
+        if list_args in kwargs:
+            kwargs[list_args] = kwargs[list_args].split(',')
```

Comment:
Does order matter for these arguments? It doesn't look like it does. If this is the case, it would probably be a good idea to use ``sorted(kwargs[list_args].split(','))`` to ensure that someone changing the order of these arguments in the comma-separated list won't result in changes to the minion (or reported changes when running in test mode).